### PR TITLE
output example for regression cbify

### PR DESCRIPTION
**Pull request by peterychang**
The output_example function assumes there's only ever 1 element in the cb_cont_label vector. It also assumes finish will be called on an example before another one is passed in
@rajan-chari do you know if these are these valid assumptions?

### CODE REVIEW

File: vowpalwabbit/cbify.cc [hunk 1]
```diff
@@ -478,9 +478,25 @@ void output_example_seq(vw& all, multi_ex& ec_seq)
...
 }
 
-void finish_example(vw& all, cbify&, example& ec)
+void output_example_regression(vw& all, cbify& data, example& ec)
+{
+  // data contains the cb_cont vector, which store among other things, loss
+  // ec contains a simple label type
+  label_data& ld = ec.l.simple;
+  const auto& cb_cont_costs= data.regression_data.cb_cont_label.costs;
+  if (cb_cont_costs.size() > 0)
```
Comment (peterychang) [inline vowpalwabbit/cbify.cc:487]: cb_cont_costs should have exactly 1 element in it (associated with the last example that predict_or_learn_regression was called with).

**Comment by peterychang** [comment]
This code seems a bit fragile since it makes several assumptions. Code is modified from the output_example in topk

File: vowpalwabbit/cbify.cc (hunk 1 above)
Comment (rajan-chari) [reply vowpalwabbit/cbify.cc:487]: Should be correct.

---
In reply to: [316680114](https://github.com/mmajzoubi/vowpal_wabbit/pull/10#discussion_r316680114) [](ancestors = 316680114)

**Comment by rajan-chari** [comment]
Should be fine.  Please merge

---
In reply to: [523909118](https://github.com/mmajzoubi/vowpal_wabbit/pull/10#issuecomment-523909118) [](ancestors = 523909118)

**Pull request merged**